js/interactive-guides/microprofile-config/microprofile-config-callback.js

```diff
@@ -82,6 +82,8 @@ var microprofileConfigCallBack = (function() {
         // correct annotation/method
         if (stepName === "ConfigureAsEnvVar") {
             __addPropToServerEnv(stepName);
+        } else if (stepName === "ConfigureViaInject") {
```

Comment:
I know it is not your changes but can you change lines 79 thru 87 to use if then else if then else if. Thanks.